profile.html

```diff
@@ -95,23 +95,23 @@ <h5 class="offcanvas-title" id="offcanvasProfileLabel">User Panel</h5>
         </div>
       </div>
     </div>
-    <main class="bg-light d-flex flex-column p-3">
+    <main class="bg-light d-flex flex-column p-3 row">
       <div class="container container-xxl col-md-7 d-flex flex-column align-items-center">
```

Comment:
```suggestion
      <div class="col-md-7">
           <div class="container container-xxl d-flex flex-column align-items-center">
```

It's better to keep container and row/col classes apart.